### PR TITLE
Fix wrong reference in ptb_word_lm.py

### PR DESCRIPTION
**Pull request by yanboliang**

### CODE REVIEW

File: tutorials/rnn/ptb/ptb_word_lm.py [hunk 1]
```diff
@@ -225,15 +225,15 @@ def make_cell():
 
     self._initial_state = cell.zero_state(config.batch_size, data_type())
     state = self._initial_state
-    # Simplified version of tensorflow_models/tutorials/rnn/rnn.py's rnn().
```
Comment (yanboliang) [inline tutorials/rnn/ptb/ptb_word_lm.py]: ```tensorflow_models/tutorials/rnn/rnn.py``` has been removed, and corresponding APIs moved to ```tensorflow/python/ops/rnn.py``` and were exported by ```tf.nn.***```.

File: tutorials/rnn/ptb/ptb_word_lm.py [hunk 2]
```diff
@@ -225,15 +225,15 @@ def make_cell():
...
+    # Simplified version of tf.nn.static_rnn().
     # This builds an unrolled LSTM for tutorial purposes only.
-    # In general, use the rnn() or state_saving_rnn() from rnn.py.
+    # In general, use tf.nn.static_rnn() or tf.nn.static_state_saving_rnn().
```
Comment (lukaszkaiser) [inline tutorials/rnn/ptb/ptb_word_lm.py]: I think it's in contrib, not in nn, no?

Comment (yanboliang) [reply tutorials/rnn/ptb/ptb_word_lm.py]: Either can work well, and ```contrib``` only import the API and implementation from ```nn```. So I think ```nn``` should be preferred. Does it reasonable?

**Comment by WeichenXu123** [comment]
Good work 💯

**Pull request merged**